torch/testing/_internal/dtype_getters.py

```diff
@@ -0,0 +1,4 @@
+"""The content of torch/testing/_dtype_getters.py should be moved here as soon as the deprecation period is over.
+"""
+
```

Comment:
Add a comment here that (because of your clever handling of deprecation warnings above) these imports will now throw a deprecation notice.

Should we create a new file for dtype_getters or stuff them in common_utils.py?

Reply:
I dislike large and unstructured files, so I would prefer an separate module. If you don't like the name, I mean there is no `common_dtype.py` yet, so this would be also an option.

Reply:
> Add a comment here that (because of your clever handling of deprecation warnings above) these imports will now throw a deprecation notice.

The whole point of `torch/testing/_dtype_getters.py` is to have access to the getters without seeing a deprecation warning. So if you import our getters from there everything is as before. Only if you import them from `torch.testing` you will see the warning.

For now we could live without `torch/testing/_internal/dtype_getters.py`, but I wanted to avoid changing all imports again after the deprecation cycle is over and everything is internalized. By adding an internal module upfront, we only need to move the functionality there in the future, but all import stay the same.

Reply:
Oh I meant that these WILL NOT THROW. Sorry, typing is hard.

Reply:
Sure; this file seems fine.